projects/observability/src/pages/trace-detail/trace-detail.page.component.ts

```diff
@@ -82,4 +92,9 @@ export class TraceDetailPageComponent {
   public onClickBack(): void {
     this.navigationService.navigateBack();
   }
+
+  public getExplorerNavigationParams = (traceDetails: ApiTraceDetails | undefined): Observable<NavigationParams> =>
```

Comment:
This is a full trace, but also doesn't appear like it can ever be called with undefined.
```suggestion
  public getExplorerNavigationParams = (traceDetails: TraceDetails): Observable<NavigationParams> =>
```

Reply:
+1